docs/standard/serialization/system-text-json-immutability.md

```diff
@@ -13,15 +13,19 @@ helpviewer_keywords:
 
 # How to use immutable types and non-public accessors with System.Text.Json
 
-In this article, you will learn how to use immutable types, such as Records, with the `System.Text.Json` namespace.
+This article shows how to use immutable types, parameterized constructors, and non-public accessors with the `System.Text.Json` namespace.
```

Comment:
Why drop the `record` term here?

Reply:
I omitted "such as Records" because this intro is shown for both 3.1 and 5.0, and "such as Records" didn't seem important enough to warrant making separate 3.1 and 5.0 intro sentences.